api/fossa/locator.go

```diff
@@ -61,9 +61,20 @@ func (l Locator) URL() string {
 	if branch == "" {
 		branch = "master"
 	}
+
+	project := l.Project
+
+	if l.Fetcher == "archive" {
+		orgID, err := GetOrganizationID()
+		if err != nil {
+			log.Warnf("Could not get OrganizationID while constructing locator")
+		}
+		project = orgID + "/" + project
```

Comment:
Huh. How did we not do this before? Was this just broken before?

Reply:
No, we never did this previously but I don't think it was really "broken" because we've never supported top-level projects with `archive` locators through the CLI until now.